app/src/main/java/org/jellyfin/androidtv/ui/playback/nextup/NextUpViewModel.kt

```diff
@@ -54,20 +59,35 @@ class NextUpViewModel(
 	private suspend fun loadItemData(id: String) = withContext(Dispatchers.IO) {
 		val item = apiClient.getItem(id) ?: return@withContext null
 
-		val thumbnail = apiClient.GetImageUrl(item, ImageOptions())
+		val thumbnail = when (userPreferences[UserPreferences.nextUpBehavior] == NextUpBehavior.EXTENDED) {
+			true -> apiClient.GetImageUrl(item, ImageOptions())
+			false -> null
+		}
 		val logo = apiClient.GetLogoImageUrl(item, ImageOptions())
 
-		val title = if (item.indexNumber != null && item.name != null)
-			"${item.indexNumber}. ${item.name}"
-		else if (item.name != null)
-			item.name
-		else ""
+		val seasonNumber = when (item.parentIndexNumber != null && item.parentIndexNumber != 0) {
+			true -> context.getString(R.string.lbl_season_number, item.parentIndexNumber)
+			false -> null
+		}
+		val episodeNumber = when {
+			(item.parentIndexNumber == 0) ->
+				context.getString(R.string.lbl_special)
```

Comment:
I'm not sure if all items with a parentIndexNumber set to 0 are specials, Did you look at other places to determine this behavior?

Reply:
This is how the web client handles it: https://github.com/jellyfin/jellyfin-web/blob/5c09077a2f12ae44e7b7d356ea8d9f35862b0b60/src/components/itemHelper.js#L21

Reply:
Thanks, in that case you should check if the item is an episode because the next up screen can show up for movies in collections/playlists.

Reply:
If the item's not an episode, the string will now be formatted as something like "12. Title". 

I couldn't figure out how to test this, though: using the playlists in the demo server, the movies always played in the music player rather than a video player, and I couldn't get collections to queue subsequent movies. I'm also not sure how to actually view/start playing from the queue that you can manually create using the "Add to Queue" button in a movie or episode's item details.

Reply:
You're right, the first branch takes care of that.